src/main/rules/GCI96/GCI96.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "DATA Avoid CSV Format",
```

Comment:
add something like that " ... and prefer Parquet format to store data into file"